serviceconnector/src/tree/ServiceConnectorGroupTreeItem.ts

```diff
@@ -4,13 +4,13 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { uiUtils } from "@microsoft/vscode-azext-azureutils";
-import { AzExtParentTreeItem, AzExtTreeItem, TreeItemIconPath, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { AzExtParentTreeItem, AzExtTreeItem, TreeItemIconPath, createContextValue, nonNullValue } from "@microsoft/vscode-azext-utils";
 import * as vscode from 'vscode';
 import { getIconPath } from "./IconPath";
 import { ServiceConnectorTreeItem } from "./ServiceConnectorTreeItem";
 
 export class ServiceConnectorGroupTreeItem extends AzExtParentTreeItem {
-    constructor(parent: AzExtParentTreeItem, public readonly resourceId: string) {
+    constructor(parent: AzExtParentTreeItem, public readonly resourceId: string, public readonly contextValuesToAdd: string[]) {
```

Comment:
Does this need to be public?

Reply:
I was following what was done in the appService package but I don't think it technically needs to.

Reply:
I'd make it private since it isn't used anywhere else.